packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -59,13 +56,31 @@ export interface ITree extends IChannel {
 	 * Additionally, once out of schema content adapters are properly supported (with lazy document updates),
 	 * this initialization could become just another out of schema content adapter and this initialization is no longer a special case.
 	 */
+	viewWith<TRoot extends ImplicitFieldSchema>(config: TreeConfiguration<TRoot>): TreeView<TRoot>;
+
+	/**
+	 * Returns a {@link TreeView} using the provided schema.
+	 * If the stored schema is compatible with the view schema specified by `config`,
+	 * the returned {@link TreeView} will expose the root with a schema-aware API based on the provided view schema.
+	 * If the provided schema is incompatible with the stored schema, the view will instead expose a status indicating the incompatibility.
+	 *
+	 * @remarks
+	 * If the tree is uninitialized, it will be implicitly initialized by this function.
+	 *
+	 * Note that other clients can modify the document at any time, causing the view to change its compatibility status: see {@link TreeView.events} for how to handle invalidation in these cases.
+	 *
+	 * Only one schematized view may exist for a given ITree at a time.
+	 * If creating a second, the first must be disposed before calling `schematize` again.
+	 * @deprecated - Replaced by {@link ITree.viewWith}. Use that method instead. Note that `viewWith` does not implicitly initialize the tree:
```

Comment:
This method will be removed; only API on `TreeView` will be `viewWith`, and it won't take an initial tree (nor will it implicitly initialize in some cases, that's done explicitly now).

It's around for now so that I could get the sample app working without letting changes sprawl any further.